lightning/src/onion_message/messenger.rs

```diff
@@ -753,7 +753,8 @@ where
 			},
 			Ok(SendSuccess::BufferedAwaitingConnection(node_id)) => {
 				log_trace!(
-					self.logger, "Buffered onion message waiting on peer connection {}: {}",
+					WithContext::from(&self.logger, Some(*node_id), None),
+					"Buffered onion message waiting on peer connection {}: {}",
 					log_suffix, node_id
 				);
```

Comment:
Would it be worth including a node id for the other cases? Not sure what would be most consistent, but we have the `destination` in case of an error and the `path` for success, though the latter is consumed.

Reply:
Yeah, but at least for the `SendSuccess::Buffered` case, the `path` that is consumed in `and_then` would start at the peer.